adserver/templates/adserver/advertiser/flight-list.html

```diff
@@ -18,6 +18,16 @@
 <h1>{% block heading %}{% trans "Manage advertising flights" %}{% endblock heading %}</h1>
 
 
+<aside class="col text-right">
+  {% if request.user.is_staff %}
+    <a href="{% url 'flight_create' advertiser.slug %}" class="btn btn-sm btn-outline-secondary mb-3" role="button" aria-pressed="true">
+      <span class="fa fa-plus mr-1" aria-hidden="true"></span>
+      <span>{% trans 'Create flight' %}</span>
```

Comment:
Should the UI make it more obvious this is an admin-only feature currently? Maybe a specific color or lock icon or something?

Reply:
Longer term, I do think we'll be opening these up to users but maybe an indication isn't bad. I'll see what I can do.

Reply:
I added a few indicators -- both color and icon.